pandas/core/frame.py

```diff
@@ -21,6 +21,7 @@
     Sequence,
 )
 import functools
+import pandas as pd
```

Comment:
You can remove this import, line 181 imports `Series` already

Reply:
Will do.